### PR TITLE
Align text left of paragraph 'Why Recruit at IIIT Kalyani? ' Section

### PR DESCRIPTION
**Pull request by pavankalyan-codes**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -346,7 +346,7 @@ h2 {
 .whyus-cont {
   background: #f6f6fa;
   padding: 80px;
-  text-align: center;
+  text-align: left;
```
Comment (pavankalyan-codes) [inline css/main.css:349]: Made text-align to left for the section

File: css/main.css [hunk 2]
```diff
@@ -357,6 +357,7 @@ h2 {
   border: 2px solid rgb(15, 70, 158);
   border-radius: 6px;
   padding: 8px;
+  text-align: center;
```
Comment (pavankalyan-codes) [inline css/main.css:360]: Made text-align of headings are center

**Comment by pavankalyan-codes** [comment]
The above 2 changes are the only changes done. everything else is prettier formatting

**Comment by pavankalyan-codes** [comment]
#36

**Review by mohit355 — APPROVED** [review]

**Pull request merged**